DEVELOPMENT.md

```diff
@@ -202,7 +205,7 @@ Some credentials in this pipeline are "compound" credentials that use the pipeli
 --- | --- | --- |
 |**`((deploy-env))-cf-username`**|The deployment environments CloudFoundry deployer username based on the instanced pipeline|:white_check_mark:|
```

Comment:
Either "development environment's" or "development environment".

Reply:
Skipping these because they aren't part of this PR (will catch on a larger docs review)